### DISCUSSION TITLE
Scope Wallet and Phantom Wallet Collected Mnemonic with Sentry

### DISCUSSION THREAD

**Discussion by BboyAwey** [getsentry/support#62] [Support] [answered]
[solana hack blamed slope mobile wallet exploit](https://decrypt.co/106680/solana-hack-blamed-slope-mobile-wallet-exploit)
[Solana 公链大规模盗币事件的分析](https://web3caff.com/zh/archives/25399)

Is there any infomation that sentry development team could provide for this mnemonic collection? Like why sentry collect mnemonic and how dose it do this? Is it unsafe?

**Comment by byehero** [comment]
Surprised! This should be a serious concern for we all developers.

**Comment by definewbee** [comment]
Whether the mnemonic should be desensitized or be ignored through some configuration?

**Comment by dcramer** [comment, marked as answer]
This is not Sentry collecting this information per-se, but it is being passed to a navigation trigger which we our SDKs may be recording (as far as I can tell). We cannot comment on _why_ its being done this way within a customers application, but for identifiable secrets we implement various safeguards to scrub them out. In this case it is not an identifiable secret other than the 'private key' nomenclature which we're adding to our detections.

We do already provide numerous ways to remove this kind of data, but the best bet is always to avoid sending it if and when possible: https://docs.sentry.io/product/data-management-settings/scrubbing/advanced-datascrubbing/

**Reply by BboyAwey** [reply]
Great to hear the official voice. That will make us feel safer when using sentry. Thanks a lot.

**Reply by definewbee** [reply]
great！
